src/bodyElement.ts

```diff
@@ -224,45 +225,47 @@ const parse = (context: Context, atoms?: Atoms) =>
             const { iframeUrl, alt } = element.interactiveTypeData ?? {};
 
             if (!iframeUrl) {
-                return new Err('No iframeUrl field on interactiveTypeData');
+                return err('No iframeUrl field on interactiveTypeData');
             }
 
-            return new Ok({ kind: ElementKind.Interactive, url: iframeUrl, alt });
+            return ok({ kind: ElementKind.Interactive, url: iframeUrl, alt });
         }
 
         case ElementType.RICH_LINK: {
             const { url, linkText } = element.richLinkTypeData ?? {};
 
             if (!url) {
-                return new Err('No "url" field on richLinkTypeData');
+                return err('No "url" field on richLinkTypeData');
             } else if (!linkText) {
-                return new Err('No "linkText" field on richLinkTypeData');
+                return err('No "linkText" field on richLinkTypeData');
             }
 
-            return new Ok({ kind: ElementKind.RichLink, url, linkText });
+            return ok({ kind: ElementKind.RichLink, url, linkText });
         }
 
         case ElementType.TWEET: {
             const { id, html: h } = element.tweetTypeData ?? {};
 
             if (!id) {
-                return new Err('No "id" field on tweetTypeData')
+                return err('No "id" field on tweetTypeData')
             } else if (!h) {
-                return new Err('No "html" field on tweetTypeData')
+                return err('No "html" field on tweetTypeData')
             }
 
-            return tweetContent(id, context.docParser(h))
-                .fmap(content => ({ kind: ElementKind.Tweet, content }));
+            return pipe(
+                tweetContent(id, context.docParser(h)),
+                rmap(content => ({ kind: ElementKind.Tweet, content })),
```

Comment:
rmap?

Reply:
could we just use map?

Reply:
`rmap` was my way of specifying `Result.map`, as `map` was already taken in this scope by `Option.map`.